HW_2/task_1.py

```diff
@@ -0,0 +1,20 @@
+"""
+Напишите программу, которая принимает на вход вещественное число и показывает сумму его цифр.
+
+Ввод: значение типа <float>
+Вывод: значение типа <int>
+
+Примеры:
+6782.0
+23
+
+0.56
+11
+"""
+num = float(input('Введите число: '))
+str_num = str(num)
+sum = 0
+for i in range(len(str_num)):
+    if str_num[i].isdigit():
+        sum += int(str_num[i])
```

Comment:
А зачем использовать функцию `range`? Почему нельзя пройтись напрямую по элементам строки?
```suggestion
for i in str_number:
    if i.isdigit():
        sum += int(i)
```
Обычно в этом случае используют  другую переменную в цикле.
```suggestion
for el in str_number:
    if el.isdigit():
        sum += int(el)
```